hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -321,6 +330,55 @@ func (mh *Memcached) implementingAPI() {
 	pkg.CheckError("updating sample", err)
 }
 
+// implementingMonitoring will customize monitoring
+func (mh *Memcached) implementingMonitoring() {
+	err := os.MkdirAll(filepath.Join(mh.ctx.Dir, "monitoring/metrics"), os.ModePerm)
```

Comment:
why have the `metrics/` folder under `monitoring/` and not create `metrics.go` under `monitoring/`?

Reply:
I agree

Reply:
Is all implementation specific to the controller? So, IMO it should leave with the controller.
Why a directory for that?

Reply:
Done